nvflare/private/fed/app/simulator/simulator.py

```diff
@@ -48,5 +47,7 @@ def parse_args():
 
     simulator = SimulatorRunner(args)
     if simulator.setup():
-        simulator.run()
-    os._exit(0)
+        run_status = simulator.run()
+    else:
+        run_status = 1
```

Comment:
should we log "setup" fail here so people know this exit code of 1 means setup failed.

Reply:
Detail error message already in the log. We will document the error code.